getusermedia.html

```diff
@@ -299,14 +299,14 @@ <h2>{{MediaStream}}</h2>
       consumer.</p>
       <p>A {{MediaStream}} object is said to be <dfn data-dfn-for="stream" id=
       "stream-active">active</dfn> when it has at least one
-      {{MediaStreamTrack}} that has not [= track/ended =]. A {{MediaStream}} that does not
-      have any tracks or only has tracks that are [= track/ended =]
+      {{MediaStreamTrack}} that has not [= MediaStreamTrack/ended =]. A {{MediaStream}} that does not
```

Comment:
Nit, global, no need for space after "[= ": 

```suggestion
      {{MediaStreamTrack}} that has not [=MediaStreamTrack/ended=]. A {{MediaStream}} that does not
```